pyproject.toml

```
@@ -5,7 +5,7 @@
 requires = [
   "setuptools == 68.1.0",
   "setuptools_scm[toml] == 7.1.0",
-  "frequenz-repo-config[api] == 0.6.2",
+  "frequenz-repo-config[api] == 0.7.0",
```

There is a but in repo-config v0.7.5, it is using an older release, this should be 0.7.5.
```suggestion
  "frequenz-repo-config[api] == 0.7.5",
```